presto-iceberg/src/test/java/com/facebook/presto/iceberg/procedure/TestRollbackToTimestampProcedure.java

```diff
@@ -71,7 +71,7 @@ public Object[][] timezones()
 
     private void createTable()
     {
-        assertUpdate("CREATE TABLE " + ICEBERG_CATALOG + "." + TEST_SCHEMA + "." + TEST_TABLE + " (id integer, value varchar)");
+        assertUpdate("CREATE TABLE IF NOT EXISTS " + ICEBERG_CATALOG + "." + TEST_SCHEMA + "." + TEST_TABLE + " (id integer, value varchar)");
```

Comment:
I think the core reason for this issue is that we operated on tables with the same table name in a non single threaded environment. So it seems that we cannot solve the issue in this way, perhaps we should either mark the test class as single threaded or make sure that all concurrently executed test cases use different table names.

Reply:
> I think the core reason for this issue is that we operated on tables with the same table name in a non single threaded environment.
I missed the point about we are using same table name in this particular test. Would it be better to use different table names instead like other tests instead of making class single threaded?

Reply:
> Would it be better to use different table names instead like other tests instead of making class single threaded?

Agree, seems this would be better.